src/Schema/PlasterManifest-v1.xsd

```diff
@@ -175,6 +175,11 @@
           <xs:documentation>The label for the choice that is presented in the user interface. The &amp; character must be respresented with the entity ref &amp;amp;</xs:documentation>
         </xs:annotation>
       </xs:attribute>
+      <xs:attribute name="condition" type="ptd:condition">
```

Comment:
A conditional choice is cool (not something I had thought of before) but I don't see that you are checking this condition where `choice|multichoice` is being processed.  BTW sorry about that other PR mixup with @bgelens, I should have had that accepted a while back.  Thanks for your work on it!   If you're running out of gas on this one, I can accept it and add the `choice` check but I wanted to give you a chance to wrap this one up.  :-)